core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -312,7 +312,7 @@ trait EntityComponent {
           filtersOption match {
             case None => sql""
             case Some(filters) =>
-              concatSqlActions(sql"#$prefix ", reduceSqlActionsWithDelim(filters, sql" and "))
+              concatSqlActions(sql"#$prefix (", reduceSqlActionsWithDelim(filters, sql" #${FilterOperators.toSql(entityQuery.filterOperator)} "), sql") ")
```

Comment:
What's the effect of the added parenthesis in the sql?

Reply:
With the parens, the SQL for the `or` version of the query looks something like:

```
SELECT *
FROM   entity e
       LEFT OUTER JOIN entity_attribute_90_93 a
                    ON a.owner_id = e.id
                       AND a.deleted = e.deleted
       LEFT OUTER JOIN entity e_ref
                    ON a.value_entity_ref = e_ref.id
       JOIN (SELECT e.id,
                    e.name
             FROM   entity e
             WHERE  e.deleted = 'false'
                    AND e.entity_type = ?
                    AND e.workspace_id = ?
                    AND ( Concat(e.name, ' ', e.all_attribute_values) LIKE 'term1'
                           OR Concat(e.name, ' ', e.all_attribute_values) LIKE 'term2'
                        )
             ORDER  BY name ASC
             LIMIT  10 offset 0) p
         ON p.id = e.id
ORDER  BY p.name ASC 
```

Without the parents, the `WHERE` clause would be:

```
             WHERE  e.deleted = 'false'
                    AND e.entity_type = ?
                    AND e.workspace_id = ?
                    AND Concat(e.name, ' ', e.all_attribute_values) LIKE 'term1'
                    OR Concat(e.name, ' ', e.all_attribute_values) LIKE 'term2'
```

The latter would match on too many rows because it'd succeed for anything in the entire table as long as it had "term2" in the all_attribute_values column, regardless of which workspace it was in or which entity type it was.